packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -544,6 +600,15 @@ export class DeltaManager
             throw new Error("Container is not attached");
         }
 
+        if (docService.policies?.storageOnly === true) {
```

Comment:
What would be the reason not to baked this code into driver itself?
This would eliminate completely notions of policies?.storageOnly and ReadOnlyType.StorageOnly at that layer. The need for policy would be gone completely (other than just FYI kind of notification from the driver).
The only complications that I see is NoDeltaStream would be included into multiple drivers.

Reply:
i'm not sure how we would disambiguate this  from other readonly cases then. seems like either way their needs to be something exposed at some layer. additionally, eventually i'd like to remove NoDetlaStream, and just support and undefined connection, but that seemed like too much churn for now